### DISCUSSION TITLE
Need documentation of HTML filtering in GFM

### DISCUSSION THREAD

**Discussion by Artoria2e5** [community/community#58528] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

GitHub currently has very weak documentation on exactly what tags are allowed to appear on the site. Like, an exhaustive list.

* It's not in https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/quickstart-for-writing-on-github, or anywhere under that tree.
* https://github.com/github/markup mentions that "aggressive sanitiz[ation]" is done, but does not actually call on any code to do the filtering, except in tests, which goes to `html-pipeline`. But `html-pipeline`'s README says github no longer uses it.
* The unofficial https://gist.github.com/seanh/13a93686bf4c2cb16e658b3cf96807f2 list is based off `html-pipeline`.

So what exactly is GitHub using to filter the HTML these days? And where on Earth can I see its source, specifically the part with the allowlist written out, and potentially open an issue *there* to ask for something to be allowed?

**Comment by milahu** [comment]
wild guess: [html_pipeline/sanitization_filter.rb](https://github.com/gjtorikian/html-pipeline/blob/main/lib/html_pipeline/sanitization_filter.rb) was ported from ruby to rust, and now its closed source.

the result should be the same, and we can always reverse-engineer the filter by brute force = by trying all kinds of html code and observing the result.

**Reply by Artoria2e5** [reply]
I tried feeding `<small>`, which is part of the filter, through, but as you can see... <small>it does not work</small>.
